src/Uno.UI/UI/Xaml/UIElement.Pointers.cs

```diff
@@ -311,23 +311,44 @@ internal HitTestability GetHitTestVisibility()
 			var that = (UIElement)sender.Owner;
 			var src = PointerRoutedEventArgs.LastPointerEvent?.OriginalSource as UIElement ?? that;
 
+			// Note: ManipulationStarting is raised in WinUI, even if RedirectWithManipulation was called.
 			that.SafeRaiseEvent(ManipulationStartingEvent, new ManipulationStartingRoutedEventArgs(src, that, args));
+
+			if (that.Visual.RedirectedPointerId == args.Pointer.Id)
```

Comment:
Why is this handling is done here? I would suspect that we should do it directly in the `InputManager.PointerManager` (and maybe the `IUnoCorePointerInputSource`). I would suspect that once a pointer is being capture by the `InteractionTracker` events are no longer bubbling in the visual tree.